axon-server-connector/src/main/java/org/axonframework/axonserver/connector/AxonServerConfiguration.java

```diff
@@ -167,6 +167,11 @@ public class AxonServerConfiguration {
      */
     private int queryThreads = 10;
 
+    /**
+     * The number of threads handling query responses. Defaults to {@code 10} threads.
```

Comment:
Doc says it defaults to 10, code says 5. I do think 5 is fine as a default, though.

Reply:
Fixed